openassessment/templates/openassessmentblock/peer/oa_peer_closed.html

```diff
@@ -29,9 +28,9 @@
 <div class="ui-slidable__content" aria-labelledby="oa_peer_{{ xblock_id }}" id="oa_peer_{{ xblock_id }}_content">
     <div class="wrapper--step__content">
         <div class="step__message message message--incomplete">
-            <h3 class="message__title">{% trans "The Due Date for This Step Has Passed" %}</h3>
+            <h5 class="message__title">{% trans "Status" %}</h5>
             <div class="message__content">
-              <p>{% trans "This step is now closed. You can no longer complete peer assessments or continue with this assignment, and you'll receive a grade of Incomplete." %}</p>
+              <p>{% trans "The Due Date for This Step Has Passed. This step is now closed. You can no longer complete peer assessments or continue with this assignment, and you'll receive a grade of Incomplete." %}</p>
```

Comment:
The titles that have been moved into the message body should be sentence case: "The due date for this step has passed."